treeherder/etl/perf.py

```diff
@@ -160,8 +162,9 @@ def _load_perf_datum(job, perf_datum):
             subtest_signature_hash = _get_signature_hash(subtest_properties)
             value = list(subtest['value'] for subtest in suite['subtests'] if
                          subtest['name'] == subtest_properties['test'])
+            application = _get_application_name(perf_datum)
```

Comment:
I believe a single call for this is enough. Though we need to be careful when moving this line of code.